common/src/main/scala/no/ndla/common/errors/ExceptionLogHandler.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Part of NDLA backend.common
```

Comment:
Vi har vel det slik overalt ellers.
```suggestion
 * Part of NDLA common
```